share/keepassxc.ini

```diff
@@ -34,6 +34,7 @@ lockdatabaseidlesec=240
 lockdatabaseminimize=false
 lockdatabasescreenlock=true
 passwordscleartext=false
+passwordemptynodots=false
```

Comment:
I actually think this should be enabled by default, or maybe even not an option, just normal behavior. There is no security implications (shoulder surfing) for which entries have or have not a password.

Reply:
I've set the option to show an empty password field to be true by default now. I didn't remove the option to toggle the functionality, I believe it can be of use when you want to obfuscate what kind of entry it is, in the event of shoulder surfing. For example, storing an ID number without a password could possibly make it more obvious that it's an ID number, and not authentication credentials. Additionally, I don't really see the harm in keeping at as an option--it might apply to certain use cases.